js-foundations.js

```diff
@@ -1,12 +1,41 @@
+var text = 'outside'
+function logIt(){
+  console.log(text)
+  var text = 'inside'
+}
+logIt()
+
+
 // ASSESSMENT 2: JAVASCRIPT FOUNDATIONS
 // Coding practical questions
 
-// 1. Write an arrow function that takes a number as an argument and decides if the number is evenly divisble by three or not. Use the test cases 15, 0, 385, -7. Expected output: "15 is divisible by three", "0 is divisible by three", "385 is not divisble by three", "-7 is not divisble by three".
-
+// 1. Write an arrow function that takes a number as an argument and decides if the number is evenly divisible by three or not. Use the test cases 15, 0, 385, -7. Expected output: "15 is divisible by three", "0 is divisible by three", "385 is not divisble by three", "-7 is not divisble by three".
+    //  function evenlyByThree(num) {
+    //      if (num % 3 === 0){
+    //          return num + " is evenly divisible by three." ;
+    //      } else {
+    //          return num + " is not evenly divisible by three." ;
+    //      }
+    //  }
+     
+    const evenlyByThree = num => (num % 3 === 0) ? num + " is evenly divisible by three." : num + " is not evenly divisible by three.";
+     
+     
+     console.log(evenlyByThree(385));
 
 
 // 2. Create an object called helloMe. Include your firstName, lastName and TWO OTHER properties of you. Add a method to your object that will return a sentence using the object properties to describe you.
 
+let helloMe = {
+        firstName: "Sebastian",
+        lastName: "Becerra",
+        languagesSpoken: ["English", "Spanish"],
+        currentlyWatching: ["Mad Men", "Parks & Rec", "Etc."],
+        getData: function() {return "My name is " + this.firstName + " " + this.lastName + " and I can speak " + this.languagesSpoken[0] + " as well as " + this.languagesSpoken[1] + "."} 
```

Comment:
Would template literals work well with object keys? 

Looked into it and it seems that it would make this a bit more complicated: 
https://stackoverflow.com/questions/33194138/template-string-as-object-property-name